Controller/Receipt/Index.php

```diff
@@ -10,13 +10,40 @@
 use Op\Checkout\Model\TransactionSuccessException;
 use Op\Checkout\Model\ReceiptDataProvider;
 
+/**
+ * Class Index
+ * @package Op\Checkout\Controller\Receipt
```

Comment:
"package" line in comment should be removed. It throws wanting when publishing in Marketplace